pagopa/io-web-profile
# [IOPID-1234] : Login L3 Container Mobile Margin

## Short description
This PR adjusts the margins of the login container on the page `/ripristino-accesso/accedi-livello-3` to match the Figma design as specified in the [Jira task.](https://pagopa.atlassian.net/browse/IOPID-1234)

## How to test
Launch the project, go to the page `/ripristino-accesso/accedi-livello-3`, and verify that on the **mobile** version, the margins of the login container match the [Figma design.](https://www.figma.com/file/2BZhyc0qxN31sF6npvbeTh/IO-Esco?type=design&node-id=534-27903&mode=design&t=DN77Q83uS57JNAaf-4)
